app/models/case_attachment_upload_group.rb

```diff
@@ -38,6 +38,6 @@ def <=>(other)
 private
 
   def get_time(upload_group)
```

Comment:
One final thought is that you could change this param name to `timestamp` or similar so it's clearer what is expected.